opencti-platform/opencti-graphql/src/migrations/1717060854132-max-shareable-marking-in-groups.js

```diff
@@ -48,27 +48,27 @@ export const up = async (next) => {
 
   await Promise.all(groupMaxMarkingRelationCreationsPromises);
 
-  // remove platform_data_sharing_max_markings from settings
+  // set platform_data_sharing_max_markings to null in settings
   const updateQuery = {
     script: {
-      params: { null: null },
-      source: 'ctx._source.platform_data_sharing_max_markings = params.null',
+      params: { fieldToRemove: 'platform_data_sharing_max_markings' },
+      source: 'ctx._source.remove(params.fieldToRemove)',
     },
     query: {
       bool: {
         must: [
           { term: { 'entity_type.keyword': { value: 'Settings' } } },
+          { exists: { field: 'platform_data_sharing_max_markings' } },
```

Comment:
exists condition doesn't match null or [] values, so I think it should be removed

Reply:
I'm gonna modify the condition with: field exist OR is field is null